### PR TITLE
Edit for clarity and spelling

### PR DESCRIPTION
**Pull request by mr-islam**
I was following along with this amazing guide to set up a new Sveltekit 2 project, and tried to make edits and suggestions for anything I thought needs more clarity

I'm sorry I didn't have time to separate each edit into a separate commit, but I commented to explain the edits that need explanation

### CODE REVIEW

**Comment by mr-islam** [comment]
I have read the CLA Document and I hereby sign the CLA

File: inlang/guides/build-a-global-svelte-app/build-a-global-svelte-app.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ export default defineConfig({
 })
 ```
 
-With the Vite-Plugin added, you can also remove the `paraglide-js compile` from your package.json, if you added those. The plugin will take care of that.
```
Comment (mr-islam) [inline inlang/guides/build-a-global-svelte-app/build-a-global-svelte-app.md:80]: I found that the `paraglide-js compile` command is added automatically during `init`, so it's there even when the use doesn't add it

**Comment by samuelstroschein** [comment]
Thank you @mr-islam! I pinged @LorisSigrist to review the changes

**Comment by LorisSigrist** [comment]
This is fantastic! Thank you very much

**Pull request merged**